### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by shinde-darshan06**

### CODE REVIEW

File: python-practice-codes/Fibonacci series.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+for i in range(0, num):
+    if i<1:
+        result = i
+    else:
+        result = first + second
+        first, second = second, result
+    lst.append(result)
+    
+print("fibonacci series are: ", lst)
```
Comment (daidipya-thakare) [inline python-practice-codes/Fibonacci series.py:15]: Please try same solution with recursion

File: python-practice-codes/first n prime numbers.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+while True:
+    is_prime=True
+    for i in range(2, num1//2+1):
+        if num1%i == 0:
+            is_prime = False
+            break
+    if is_prime == True:
+        print(num1)
+        count = count+1
+        #num1 = num1+1
```
Comment (daidipya-thakare) [inline python-practice-codes/first n prime numbers.py]: Please remove commented code

File: python-practice-codes/given number is prime or not.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Write a program in Python to check given number is prime or not.
+
+
+num = int(input("enter number: ")
+if num<=1:
+    print("{} is not prime number".format(num))
+else:
+    is_prime = True
+    for i in range(2,num//2+1):
```
Comment (daidipya-thakare) [inline python-practice-codes/given number is prime or not.py:9]: Please add one more solution to this with more optimized version of  this code using log function.

**Review by daidipya-thakare — CHANGES_REQUESTED** [review]

File: python-practice-codes/Fibonacci series.py (hunk 1 above)
Comment (shinde-darshan06) [reply python-practice-codes/Fibonacci series.py:15]: done

File: python-practice-codes/given number is prime or not.py (hunk 3 above)
Comment (shinde-darshan06) [reply python-practice-codes/given number is prime or not.py:9]: done

**Review by shinde-darshan06 — COMMENTED** [review]
all changes done.

**Review by daidipya-thakare — APPROVED** [review]
ok

**Pull request merged**